### ISSUE TITLE
Does the physics-informed loss function participate in backpropagation?

### ISSUE THREAD

**Issue by wanzhe45** [fangyuanmao/PID#46]
Thank you for your excellent work. I do have a small question, though. After reviewing your code, I noticed that the losses returned in get_loss_rec and get_loss_tev seem to have requires_grad=False. How, then, do they still provide physics-guided effects when generating infrared images?

**Comment by fangyuanmao** [comment]
The gradient of TeVNet is needless, thus we set the requires_grad of TeVNet is false. However, the outputs of LDM is used to calculate the loss, the gradient of the loss can be back-propagated.
